config/spaceruinblacklist.txt

```diff
@@ -40,4 +40,4 @@
 #_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
 #_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
 #_maps/RandomRuins/SpaceRuins/forgottenship.dmm
-
+#_maps/RandomRuins/SpaceRuins/corporate_mining.dmm
```

Comment:
While I appreciate this, this is unneccessary at the moment, as we plan to make a ruin database instead.